mr/src/test/java/org/elasticsearch/hadoop/serialization/CommandTest.java

```diff
@@ -64,43 +65,55 @@
-                { ConfigurationOptions.ES_OPERATION_UPDATE, true, EsMajorVersion.V_1_X },
-                { ConfigurationOptions.ES_OPERATION_INDEX, false, EsMajorVersion.V_2_X },
-                { ConfigurationOptions.ES_OPERATION_CREATE, false, EsMajorVersion.V_2_X },
-                { ConfigurationOptions.ES_OPERATION_UPDATE, false, EsMajorVersion.V_2_X },
-                { ConfigurationOptions.ES_OPERATION_INDEX, true, EsMajorVersion.V_2_X },
-                { ConfigurationOptions.ES_OPERATION_CREATE, true, EsMajorVersion.V_2_X },
-                { ConfigurationOptions.ES_OPERATION_UPDATE, true, EsMajorVersion.V_2_X },
-                { ConfigurationOptions.ES_OPERATION_INDEX, false, EsMajorVersion.V_5_X },
-                { ConfigurationOptions.ES_OPERATION_CREATE, false, EsMajorVersion.V_5_X },
-                { ConfigurationOptions.ES_OPERATION_UPDATE, false, EsMajorVersion.V_5_X },
-                { ConfigurationOptions.ES_OPERATION_INDEX, true, EsMajorVersion.V_5_X },
-                { ConfigurationOptions.ES_OPERATION_CREATE, true, EsMajorVersion.V_5_X },
-                { ConfigurationOptions.ES_OPERATION_UPDATE, true, EsMajorVersion.V_5_X },
-                { ConfigurationOptions.ES_OPERATION_INDEX, false, EsMajorVersion.V_6_X },
-                { ConfigurationOptions.ES_OPERATION_CREATE, false, EsMajorVersion.V_6_X },
-                { ConfigurationOptions.ES_OPERATION_UPDATE, false, EsMajorVersion.V_6_X },
-                { ConfigurationOptions.ES_OPERATION_INDEX, true, EsMajorVersion.V_6_X },
-                { ConfigurationOptions.ES_OPERATION_CREATE, true, EsMajorVersion.V_6_X },
-                { ConfigurationOptions.ES_OPERATION_UPDATE, true, EsMajorVersion.V_6_X },
-                { ConfigurationOptions.ES_OPERATION_INDEX, false, EsMajorVersion.V_7_X },
-                { ConfigurationOptions.ES_OPERATION_CREATE, false, EsMajorVersion.V_7_X },
-                { ConfigurationOptions.ES_OPERATION_UPDATE, false, EsMajorVersion.V_7_X },
-                { ConfigurationOptions.ES_OPERATION_INDEX, true, EsMajorVersion.V_7_X },
-                { ConfigurationOptions.ES_OPERATION_CREATE, true, EsMajorVersion.V_7_X },
-                { ConfigurationOptions.ES_OPERATION_UPDATE, true, EsMajorVersion.V_7_X },
-                { ConfigurationOptions.ES_OPERATION_INDEX, false, EsMajorVersion.V_8_X },
-                { ConfigurationOptions.ES_OPERATION_CREATE, false, EsMajorVersion.V_8_X },
-                { ConfigurationOptions.ES_OPERATION_UPDATE, false, EsMajorVersion.V_8_X },
-                { ConfigurationOptions.ES_OPERATION_INDEX, true, EsMajorVersion.V_8_X },
-                { ConfigurationOptions.ES_OPERATION_CREATE, true, EsMajorVersion.V_8_X },
-                { ConfigurationOptions.ES_OPERATION_UPDATE, true, EsMajorVersion.V_8_X }
+        return Arrays.asList(new Object[][]{
+                {ConfigurationOptions.ES_OPERATION_INDEX, false, EsMajorVersion.V_1_X},
```

Comment:
Nit: Thanks for adding these test cases, but can you revert the spacing changes on unaffected lines? Would help keep history tidy.

Reply:
Or better yet, these could probably be generated with a couple loops.

Reply:
I changed generation with loops. The array with boolean values is here to keep exactly the same order as before in the returned list